README.md

```diff
@@ -1,54 +1,78 @@
 # The Odin Project
+![odin logo](app/assets/images/logo.svg)
 
-[The Odin Project](https://www.theodinproject.com/) (TOP) is an open-source curriculum for learning full-stack web development. Our mission is to provide a comprehensive curriculum to learn web development for free. We help users learn the skills and build the impressive portfolio of projects they need to get hired as a web developer.
+[The Odin Project](https://www.theodinproject.com/) (TOP) is an open-source curriculum for learning full-stack web 
+development. Our mission is to provide a comprehensive curriculum to learn web development for free. We help users learn 
+the skills and build the impressive portfolio of projects they need to get hired as a web developer.
 
-Some of the topics we cover include:
+----------
 
--   Ruby
--   Ruby on Rails
--   SQL
--   HTML and CSS
--   Javascript
--   Node
--   React
--   How to go about getting a job in the industry
+## About
 
-This repo contains the TOP app which pulls in lesson content as well as front-end and back-end code. To see the actual lesson content that gets pulled in, please go to the [TOP curriculum repo](https://github.com/TheOdinProject/curriculum).
+_TODO - flesh this out_
 
-Our community can be found on the [TOP Discord server](https://discord.gg/fbFCkYabZB).
+This repo contains the TOP app which pulls in lesson content as well as front-end and back-end code. To see the actual 
+lesson content that gets pulled in, please go to the [TOP curriculum repo](https://github.com/TheOdinProject/curriculum).
 
+## Useful Links
+- [Issue Tracker](https://github.com/TheOdinProject/theodinproject/issues) - for raising bugs or feature requests
+- [App Wiki](https://github.com/TheOdinProject/theodinproject/wiki) - for more detailed information about contributing to the app
+- [Curriculum Repo](https://github.com/TheOdinProject/curriculum) - where all the lesson content for the website is pulled from
+- [Discord Community](https://discord.gg/fbFCkYabZB) - the main meeting point for maintainers and learners
+- [Backlog / Project Board](https://github.com/orgs/TheOdinProject/projects/12/views/1) - the current roadmap for the maintainer team
+- [Community Forum](https://github.com/TheOdinProject/theodinproject/discussions) - for questions, ideas and chatting with the maintainers
+
+## Tech Stack
+The core technologies that the app runs on. This is what you'll need to get the app running locally. 
+
+- **Ruby** for the core language
+- **Node** for JavsScript bundling 
+- **Yarn** for JavaScript packages
+- **Postgres SQL** for the database
+- **Redis** for background jobs
+
+### Core Frameworks and Libraries
+- Ruby on Rails
+- Tailwind CSS
+- Webpacker / Shakapacker
+
+### Notable Tools
+- ViewComponent
+- React
+- Sidekiq
+- Kramdown
+
+### Testing and Linting
+- RSpec
+- Jest
+- Rubocop
+- Erblint
+- ESLint
+  
```

Comment:
Seeing it now I feel like this is overkill, at least for the README. Perhaps better suited to the wiki or another doc.

Reply:
I can see where you're coming from by feeling it's overkill. I can't think of another spot for it though 😅 
There would be no harm in leaving it here imo

Reply:
What about moving it to the CONTRIBUTING file under a new `Tech we use` section?